docs/resources/iotda_device_linkage_rule.md

```diff
@@ -0,0 +1,280 @@
+      - **mode**: Required, whether the device needs to reply to the confirmation message after receiving the command.
+        The default is ACK mode. `ACK` indicates that the confirmation message needs to be replied,
+        `NOACK` indicates that the confirmation message does not need to be replied.
+      - **from**: Optional, the address of the command sender.
+        When the App initiates a request, the format is /users/{userId},
+        when the application server initiates a request, the format is /{serviceName},
+        and when the IoT platform initiates a request, the format is /cloud/{serviceName}.
+      - **to**: optional, the address of the command receiver, the format is /devices/{device_id}/services/{service_id}.
+      - **method**: optional, the command name defined in the product model.
+      - **body**: optional, the message body of the command, which contains key-value pairs, each key is the parameter
+        name of the command in the product model. The specific format requires application and device conventions.
+
+<a name="IoTDA_smn_forwarding"></a>
+The `smn_forwarding` block supports:
+
+* `region` - (Required, String) Specifies the region to which the SMN belongs.
+
+* `project_id` - (Required, String) Specifies the project ID to which the SMN belongs.
+
+* `topic_name` - (Required, String) Specifies the topic name of the SMN.
+
+* `topic_urn` - (Required, String) Specifies the topic URN of the SMN.
+
+* `message_title` - (Required, String) Specifies the message title.
+
+* `message_content` - (Required, String) Specifies the message content.
+
+<a name="IoTDA_device_alarm"></a>
+The `device_alarm` block supports:
+
+* `name` - (Required, String) Specifies the name of the alarm.
+
+* `type` - (Required, String) Specifies the type of the alarm. The options are as follows:
+  + **fault**: Report alarms.
+  + **recovery**: Clear alarms.
+
+* `severity` - (Required, String) Specifies the severity level of the alarm.
+The valid values are **warning**, **minor**, **major** and **critical**.
+
+* `description` - (Optional, String) Specifies the description of the alarm.
+
+<a name="IoTDA_effective_period"></a>
+The `effective_period` block supports:
+
+* `start_time` - (Required, String) Specifies the start time, in the format: `HH:mm`.
+For example: `03:00`.
+
+* `end_time` - (Required, String) Specifies the end time, in the format: `HH:mm`.
+For example: `10:00`. If the end time is the same as the start time, the effective period is the whole day.
+
+* `days_of_week` - (Optional, String) Specifies a list of days of week, separated by commas. 1 represents Sunday,
+2 represents Monday, and so on. Defaults to `1,2,3,4,5,6,7` (every day).
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID in UUID format.
+
+## Import
+
+device linkage rules can be imported using the `id`, e.g.
```

Comment:
The first letter of the sentence needs to be capitalized.

Reply:
fix